apps/web/src/scenes/ContentPages/ContentModules/FeaturedProfiles.tsx

```diff
@@ -0,0 +1,150 @@
+import { useMemo } from 'react';
+import styled from 'styled-components';
+
+import breakpoints, { size } from '~/components/core/breakpoints';
+import { Carousel } from '~/components/core/Carousel/Carousel';
+import GalleryLink from '~/components/core/GalleryLink/GalleryLink';
+import { HStack, VStack } from '~/components/core/Spacer/Stack';
+import { BaseM, TitleXS } from '~/components/core/Text/Text';
+import transitions from '~/components/core/transitions';
+import { useBreakpoint } from '~/hooks/useWindowSize';
+import colors from '~/shared/theme/colors';
+
+import { CmsTypes } from '../cms_types';
+
+type FeaturedProfilesProps = {
+  profiles: CmsTypes.FeaturedProfile[];
+};
+
+export default function FeaturedProfiles({ profiles }: FeaturedProfilesProps) {
+  const breakpoint = useBreakpoint();
+
+  const slideContent = useMemo(() => {
+    const itemsPerSlide =
+      breakpoint === size.mobile || breakpoint === size.mobileLarge
+        ? 1
+        : breakpoint === size.tablet
+        ? 2
+        : 3;
```

Comment:
hypernit: would love to useMemo `itemsPerSlide` separately with if/else for more readability